concent_api/concent_api/settings/base.py

```diff
@@ -49,7 +49,7 @@
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
+        'DIRS': [os.path.join(BASE_DIR, 'core/templates'), os.path.join(BASE_DIR, 'core/templates/admin')],
```

Comment:
Split list into separate lines.
Use `os.path.join`.
I don't think you should add `'core/templates/admin'`. Instead of that you should use `admin/` where you use template name from this directory.

Reply:
Done.